h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -99,6 +102,35 @@ function TextField({
 }
 
 export default function CreateGroupForm() {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+
+  const createGroup = async () => {
+    try {
+      console.log('sending request');
+      const response = await fetch('http://localhost:5000/api/groups', {
+        method: 'POST',
+        body: JSON.stringify({
+          name: name,
+          description: description,
+        }),
+        credentials: 'same-origin',
```

Comment:
This is [the default](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#including_credentials), so you shouldn't need to specify it.

Reply:
Done